synapse/handlers/room_member.py

```diff
@@ -887,21 +904,26 @@ async def _is_remote_room_too_complex(self, room_id, remote_room_hosts):
             return complexity["v1"] > max_complexity
         return None
 
-    async def _is_local_room_too_complex(self, room_id):
+    async def _is_local_room_too_complex(self, room_id: str) -> bool:
         """
         Check if the complexity of a local room is too great.
 
         Args:
-            room_id (str)
-
-        Returns: bool
+            room_id
```

Comment:
This line seems useless now that it doesn't even have the type. Could you please remove it or add a documentation about this parameter?